tests/plugins/save/save.js

```diff
@@ -1,9 +1,31 @@
 /* bender-tags: editor */
-/* bender-ckeditor-plugins: toolbar,save,wysiwygarea */
+/* bender-ckeditor-plugins: toolbar,save,wysiwygarea,sourcearea */
 
 bender.test( {
-	'test save event': function() {
-		var editor = CKEDITOR.replace( 'editor' ),
+	'test save event in WYSIWYG mode': function() {
```

Comment:
As code for tests is nearly identical, I'd think about creating some function to generate them or just to move common logic into it.